ods_ci/tasks/Resources/RHODS_OLM/uninstall/codeflare_uninstall.resource

```diff
@@ -0,0 +1,57 @@
+*** Settings ***
+Documentation    RHODS CodeFlare operator uninstallation keywords
+Resource   ../install/oc_install.robot
+Resource   ../../../../tests/Resources/Common.robot
+Resource   oc_uninstall.robot
+
+Library    Process
+
+
+*** Keywords ***
+Uninstalling CodeFlare Operator
+    [Documentation]   Uninstalls the RHODS CodeFlare operator if it is installed
+    ${is_operator_installed} =  Is CodeFlare Installed
+    IF  ${is_operator_installed}
+        Log  Uninstalling CodeFlare operator in ${cluster_type}  console=yes  AND
+        Uninstall CodeFlare
+    END
+
+Uninstall CodeFlare
+    [Documentation]   Removes the RHODS CodeFlare operator subscription and CSV
+    ${return_code}    ${output}    Run And Return Rc And Output
+    ...    oc delete subscription rhods-codeflare-operator -n openshift-operators
+    Should Be Equal As Integers	${return_code}	 0   msg=Error deleting CodeFlare subscription
+    ${return_code}    ${output}    Run And Return Rc And Output
+    ...    oc delete csv `oc get csv -n openshift-operators |grep rhods-codeflare-operator |awk '{print $1}'`
+    ...    -n openshift-operators
+    Should Be Equal As Integers	${return_code}	 0   msg=Error deleting CodeFlare CSV
```

Comment:
these might be covered by `Uninstall ISV Operator From OperatorHub Via CLI` keyword. Do you mind trying?